plugins/auth-backend/api-report.md

```diff
@@ -486,7 +486,7 @@ export const providers: Readonly<{
       signIn: {
         resolver: SignInResolver<CloudflareAccessResult>;
       };
-      cache?: CacheClient | undefined;
+      cache?: CacheService | undefined;
```

Comment:
Since this interface grew, it's kinda technically breaking isn't it :) I know it's originally a service interface and we have more lax rules for those in general, but it did affect our own tests code after all.